docs/sql-migration-guide.md

```diff
@@ -22,6 +22,11 @@ license: |
 * Table of contents
 {:toc}
 
+## Upgrading from Spark SQL 3.3 to 3.4
+  
```

Comment:
Is this note related tot his change? the second one is

Reply:
Thanks for point it out!, just removed the first bullet which has been merged by mistake when resolving the doc conflicts.